editor/src/com/talosvfx/talos/editor/project2/apps/SceneHierarchyApp.java

```diff
@@ -53,11 +53,7 @@ public void updateForGameAsset (GameAsset<Scene> gameAsset) {
 
 	@Override
 	public String getAppName () {
-		if (gameAsset != null) {
-			return "Hierarchy - " + gameAsset.nameIdentifier;
-		} else {
-			return "Hierarchy - ";
-		}
+		return "Hierarchy";
```

Comment:
Hierarchy should probably show the scene its editing. You can potentially have many of these